cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/validator/AzureIDBrokerObjectStorageValidatorTest.java

```diff
@@ -127,7 +127,7 @@ public void setup() {
     public void testValidateObjectStorageWithoutFileSystems() {
         SpiFileSystem fileSystem = new SpiFileSystem("test", FileSystemType.ADLS_GEN_2, null);
         ValidationResultBuilder resultBuilder = new ValidationResultBuilder();
-        underTest.validateObjectStorage(client, fileSystem, "", resultBuilder);
+        underTest.validateObjectStorage(client, fileSystem, "", null, resultBuilder);
```

Comment:
could you pls add a unit test to cover single RG case, too?

Reply:
sure